08-examples/00-overview.md

```diff
@@ -0,0 +1,28 @@
+---
+pageTitle: TypeDB Examples
+keywords: typedb, examples, migration, queries, schema
+longTailKeywords: typedb examples, typedb migration, typedb query examples, typedb schema example
+Summary: TypeDB examples showcasing schema definition, data migration and retrieval queries
+---
+
+## Examples
+
+[**Phone calls (Java, JS, Python):**](https://github.com/vaticle/typedb-examples/tree/master/phone_calls) a database of customers of a fictional telecom company and calls they make.
+
+[**XCOM 2 (Java):**](https://github.com/vaticle/typedb-examples/tree/master/xcom) a database of interdependent research tasks, featuring automatic inference of available research based on completed tasks and available items.
+
+[**Catalogue of Life (Java):**](https://github.com/vaticle/typedb-examples/tree/master/catalogue_of_life) a database containing the information about the taxonomy of life on Earth, showcasing (semi-)automatic loading of data using [TypeDB-Loader.](https://github.com/typedb-osi/typedb-loader)
```

Comment:
I would like to know what _semi-_ automatic loading is... but in seriousness, I think we should just replace this with "automatic" or "automated".

Reply:
"Automated" is miuch better.